modules/core/client/app/init.js

```diff
@@ -30,7 +30,9 @@ angular.module(ApplicationConfiguration.applicationModuleName).run(function ($ro
         if (Authentication.user !== undefined && typeof Authentication.user === 'object') {
           $state.go('forbidden');
         } else {
-          $state.go('authentication.signin');
+          $state.go('authentication.signin').then(function () {
```

Comment:
@mleanos just UX, for me it seems better to let the user know why the signin page is being displayed rather than the requested one.